install_config/syncing_groups_with_ldap.adoc

```diff
@@ -176,46 +184,59 @@ OpenShift. Your files must contain one unique group identifier per line.
 ====
 
 ----
-$ oadm groups sync --whitelist=whitelist.txt --sync-config=config.yaml --confirm
-$ oadm groups sync --blacklist=blacklist.txt --sync-config=config.yaml --confirm
-$ oadm groups sync group_unique_idenitifer --sync-config=config.yaml --confirm
-$ oadm groups sync group_unique_idenitifer   \
-                           --whitelist=whitelist.txt \
-                           --blacklist=blacklist.txt \
-                           --sync-config=config.yaml --confirm
-$ oadm groups sync --type=openshift --whitelist=whitelist.txt --sync-config=config.yaml --confirm
+$ oadm groups sync --whitelist=<whitelist_file> --sync-config=config.yaml --confirm
+$ oadm groups sync --blacklist=<blacklist_file> --sync-config=config.yaml --confirm
+$ oadm groups sync <group_unique_identifier> --sync-config=config.yaml --confirm
+$ oadm groups sync <group_unique_identifier> \
```

Comment:
@stevekuznetsov Added `<>` around the `group_unique_identifier`s as I understood them to be replaceables.